### PR TITLE
prometheus: use correct readiness endpoint

### PR DESCRIPTION
**Pull request by sylr**
#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [ ] Chart Version bumped
- [ ] Variables are documented in the README.md
- [ ] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

**Comment by zanhsieh** [comment]
@sylr 
Would you sign DCO and bump chart to latest version please? Thank you.

File: stable/prometheus/templates/pushgateway-deployment.yaml [hunk 1]
```diff
@@ -38,11 +38,7 @@ spec:
             - containerPort: 9091
           readinessProbe:
             httpGet:
-            {{- if (index .Values "pushgateway" "extraArgs" "web.route-prefix") }}
-              path: /{{ index .Values "pushgateway" "extraArgs" "web.route-prefix" }}/#/status
-            {{- else }}
-              path: /#/status
-            {{- end }}
+              path: /-/ready
```
Comment (davidkarlsen) [inline stable/prometheus/templates/pushgateway-deployment.yaml:41]: Doesn't web.route-prefix need to be taken into consideration?

Comment (sylr) [reply stable/prometheus/templates/pushgateway-deployment.yaml:41]: indeed.

**Comment by zanhsieh** [comment]
@sylr 
Could you resolve the conflicting files below on the bottom of this PR please? Thank you.

**Pull request closed without merging**